src/schemas/28-1900/schema.js

```diff
@@ -127,9 +127,21 @@ let schema = {
   ['address', 'hospitalAddress'],
   ['phone', 'daytimePhone'],
   ['phone', 'eveningPhone'],
-  ['serviceHistory']
 ].forEach((args) => {
   schemaHelpers.addDefinitionToSchema(schema, ...args);
 });
 
+const requiredServiceHistory = _.merge(_.cloneDeep(definitions.serviceHistory), {
+  minItems: 1,
+  items: {
+      required: ['serviceBranch', 'dischargeType'],
+  }
+});
+
+requiredServiceHistory.items.properties.dateRange =  _.merge(_.cloneDeep(definitions.dateRange), {
```

Comment:
On the fence about this approach- almost just included the entire object in here instead of piecing it from the different parts. Sort of wish i could pass a required argument to `addDefintionToSchema` + `definitions`. I could add another `definitions` called `requiredServiceHistory`.

Reply:
I think we should add a requiredServiceHistory definition.

Reply:
Perfect.